### PR TITLE
add z-filmz-originals

### PR DESCRIPTION
**Pull request by JaseNZC**
A request.

### CODE REVIEW

File: scrapers/zfilmzoriginals.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="info-section--title"]/text()
+      Date:
+        selector: //span[@title="Published at"]
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //span[@title="Models"]//a
```
Comment (bnkai) [inline scrapers/zfilmzoriginals.yml:17]: I would add the url to make it easier to scraper non common performers
```yaml
        URL:
          selector: //span[@title="Models"]//a/@href
          postProcess:
            - replace:
                - regex: ^
                  with: https://www.z-filmz-originals.com
```

File: scrapers/zfilmzoriginals.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      Tags:
+        Name: //span[@title="Categories"]//a
```
Comment (bnkai) [inline scrapers/zfilmzoriginals.yml:19]: some users might want `all` the tags ( the categories ones are the ones i would also add be default)
maybe add the combined selector commented out?
```
//span[@title="Categories"]//a|//div[@class="tags mt-3"]/a
```

**Comment by bnkai** [comment]
can you rename the yml file(update the list also)? It should start with a capitalized letter to make the folder sorting easier

**Comment by JaseNZC** [comment]
Sorted with some luck :-)

**Pull request merged**